atlp-rwanda/script-heroes-frontend
# #174797454 create an empty react project

#### What does this PR do?
- Creating an empty react project
#### Description of Task to be completed?
- Initialize a [react](https://reactjs.org/) app
- Configuring [webPack](https://webpack.js.org/) for bundling
- Configuring [Eslint](https://eslint.org/docs/user-guide/getting-started) for linting
#### How should this be manually tested?
- Clone this repo `git clone https://github.com/atlp-rwanda/script-heroes-frontend.git`
- Switch your working directory `cd script-heroes-frontend`
- Checkout the branch `git checkout `ch-setting-react-project-#174797454`
- Install all dependencies `yarn install`
- Start the project `yarn start`
> You should see an app running on `http://localhost:8080` in your browser
#### Any background context you want to provide?
- [yarn](https://yarnpkg.com/) is a package manager as npm do `
- If yarn is not working install it manually
- Instead of yarn you can use also npm
#### What are the relevant pivotal tracker stories?
[#174797454](https://www.pivotaltracker.com/story/show/174797454)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/44426710/98659552-e3b97180-234c-11eb-9f70-342bf54dc84e.png)

#### Questions:
